scout/server/app.py

```diff
@@ -74,7 +74,7 @@ def create_app(config_file=None, config=None):
     def check_user():
         if not app.config.get("LOGIN_DISABLED") and request.endpoint:
             # check if the endpoint requires authentication
-            static_endpoint = "static" in request.endpoint
+            static_endpoint = "static" in request.endpoint or request.endpoint == "report.report"
```

Comment:
🤣
Sure, why not! Then we fix chanjo-report in due time? I'm in if you also file an issue there!

Reply:
> Sure, why not!

It's actually different from before. Before it was `or "report" in request.endpoint`